samples/projects/application_for_backend/src/Main.qs

```diff
@@ -0,0 +1,16 @@
+import Std;
```

Comment:
This should be either `open Std;` or `import Std.*;` (or import the individual items you want to use)

Reply:
Just to add further clarification in case it isn't already known -- `import Std;` does nothing, since it brings in the "`Std`" name as "`Std`", but it is already available as that name.

Reply:
Changed to a mix of importing everything from the ContosoBackend and an individual item from Std.